aws/keycloak-backing-services/main.tf

```diff
@@ -44,6 +46,12 @@ variable "chamber_parameter_name_pattern" {
   description = "Format string for creating SSM parameter name used to store chamber parameters. The default is usually best."
 }
 
+variable "chamber_kms_key_id" {
+  type        = "string"
+  default     = "alias/aws/ssm"
```

Comment:
why `"alias/aws/ssm"`?

Reply:
If you do not specify a key for SSM operations, AWS will by default use the key id `alias/aws/ssm`